tests/test_zeitwochenmarkt.py

```diff
@@ -0,0 +1,53 @@
+from recipe_scrapers.zeitwochenmarkt import ZeitWochenmarkt
+from tests import ScraperTest
+
+# test recipe's URL
+# https://www.przepisy.pl/przepis/placki-ziemniaczane
```

Comment:
I think this comment is incorrect? Please remove or change just to avoid confusion.

Reply:
Oops.